pkg/workload/schemachange/generate_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package schemachange
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgcode"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPickOne(t *testing.T) {
+	r := rand.New(rand.NewSource(0))
+
+	v, err := PickOne(r, []int{0, 1, 2, 3, 4})
+	require.NoError(t, err)
+	require.Equal(t, 4, v)
```

Comment:
Does this line mean "the return of randomly picking one from {0,1,2,3,4} is always 4"? Why not something like "the return should be either 0 or 1 or 2 or 3 or 4?"